src/collect/cli.rs

```diff
@@ -58,6 +61,12 @@ Example: --probe tp:skb:kfree_skb --probe kprobe:consume_skb"
         default_value = "false"
     )]
     pub(super) print: bool,
+    #[arg(
+        long,
+        help = "Pretty-print the event in a given format",
```

Comment:
I find this text a bit confusing. For me it means that if you don't add this arg, events are not printed but they are, this is just an opt to select the format not to select whether to print them or not (for that we have --print). How about "--format"?

Reply:
So when we implement custom formatting we'll use `--custom-fmt "some format"`? If we use `--format` we'd need something like `--format custrom --custom-fmt "..."` or maybe just `--format "someformat"`...
Either work for me. My only concern is the verb in that sentence. For me, saying that the option "pretty-prints the event" implies that if not given, the even is not printed. Maybe something like "selects one of the predefined pretty formats for event printing" makes it a bit clearer that the event would be printed anyhow and that this option only allows you to say how.